test/unit/sqs/index.test.ts

```diff
@@ -192,28 +197,14 @@ describe("SqsQueue tests", () => {
 			await queue.sendMessage(testPayload);
 		});
 
-		it("Message deleted from the queue when unretryable", async (done: DoneCallback) => {
+		it("Message deleted from the queue when unretryable", async () => {
 
 			const testPayload = generatePayload();
 
-			const deleteMock = jest.fn();
-			queue.sqs.deleteMessage = deleteMock;
```

Comment:
So that was the reason why the test was flaky. Because we mocked the deletion function the test message wasn't removed from the queue and went back on the queue after the visibility timeout. Which is 12s. Then during integration tests which use the queue it was replayed breaking these tests